woocommerce/woocommerce-ios
# Edit Products: enable/disable product reviews in Settings

Fixes #2083 

## Description
As part of Milestone 3, we want to give to users the ability to enable or disable reviews on products. In this PR I added a new cell under the section "More Options" in Product Settings, which allows the user to change the status of the reviews for a product.

## Testing

#### Test case 1
1. Lunch the app, and navigate to a product's detail.
2. Open product settings, and edit the "Enable Reviews" flag.
3. Save the product, and make sure the changes are applied correctly, also on the web.

#### Test case 2
1. Set the `editProductsRelease3` flag to `false` in the code.
2. Lunch the app, and navigate to a product's detail.
3. Open product settings, and make sure that the "Enable Reviews" cell is not shown.

## Screenshots
| Enable Product Reviews             |  Enable Product Reviews (dark mode) |
:-------------------------:|:-------------------------:
![Simulator Screen Shot - iPhone 11 Pro - 2020-07-16 at 12 16 44](https://user-images.githubusercontent.com/495617/87660006-ad813200-c75e-11ea-821e-5fe8c9cee4ae.png) | ![Simulator Screen Shot - iPhone 11 Pro - 2020-07-16 at 12 17 00](https://user-images.githubusercontent.com/495617/87660019-b114b900-c75e-11ea-9ebe-79eccdb16f86.png)



Update release notes:
